From 935732278c8470a887029533658ef956a3acebc0 Mon Sep 17 00:00:00 2001
From: "Alam, Sahibex" <sahibex.alam@intel.com>
Date: Mon, 28 Apr 2025 14:38:59 +0000
Subject: [PATCH] Update RELEASE_PLATFORM_SECURITY_PATCH string

Security_patch_level needs to be updated
When ASB Security patches are integrated

Tracked-On: OAM-125942
Signed-off-by: Alam, Sahibex <sahibex.alam@intel.com>
---
 flag_values/bp1a/RELEASE_PLATFORM_SECURITY_PATCH.textproto | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flag_values/bp1a/RELEASE_PLATFORM_SECURITY_PATCH.textproto b/flag_values/bp1a/RELEASE_PLATFORM_SECURITY_PATCH.textproto
index 4075b588..985e7b01 100644
--- a/flag_values/bp1a/RELEASE_PLATFORM_SECURITY_PATCH.textproto
+++ b/flag_values/bp1a/RELEASE_PLATFORM_SECURITY_PATCH.textproto
@@ -1,4 +1,4 @@
 name: "RELEASE_PLATFORM_SECURITY_PATCH"
 value: {
-  string_value: "2025-03-05"
+  string_value: "2025-06-01"
 }
-- 
2.34.1

